portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Generates a greeting to the user from the DataServlet server and adds it to the DOM
+ */
+function getGreeting() {
+  fetch('/comment').then(response => response.json()).then((messages) => {
+    const messageElement = document.getElementById('message-container');
+
+    messageElement.innerHTML = '';
+    messageElement.appendChild(
+        createListElement('Message: ' + messages[0]));
+    messageElement.appendChild(
+        createListElement('Message: ' + messages[1]));
+    messageElement.appendChild(
+        createListElement('Message: ' + messages[2]));
```

Comment:
Use a for...of loop to append arbitrary number of messages.

Reply:
+1 for using for loop